include/toolchain/gcc.h

```diff
@@ -117,8 +117,11 @@ do {                                                                    \
 
 /* Using this places a function in RAM instead of FLASH.
  */
+
+#define __ramdata	__attribute__((__section__(".ramdata")))
```

Comment:
Can you explain why we need to create this section, as far as I know, data should be linked into ram by default.